src/app/core/interceptors/loading.interceptor.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpInterceptor, HttpRequest, HttpHandler, HttpEvent, HttpResponse } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { tap } from "rxjs/operators";
+import { LoaderService } from "../services/loader.service";
+
+@Injectable()
+export class LoadingInterceptor implements HttpInterceptor {
+    constructor(private loaderService: LoaderService) {}
+
+    intercept(req: HttpRequest<any>, next: HttpHandler) {
+        this.loaderService.setLoadingStatus(true);
+        return next.handle(req).pipe(
+            tap(
+                (event: HttpEvent<any>) => {
```

Comment:
Please, try to avoid any and use real types for each variable, method and property. It will make your app much easier to maintain